### PR TITLE
Disable datazones popup; refs #212

### PR DESCRIPTION
**Pull request by mvl22**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Great to see, will look at code and merge. I imagine this involves removing lots of code, as with backend code :relieved:

File: src/datasets.js [hunk 1]
```diff
@@ -473,6 +473,8 @@ const datasets = {
 	// #!# Need to define more clearly the assumed data structure, e.g. the 'charts' key shows a part field
 	charts: {
 
+		// #!# Disabled this popup as of September 2024 - can be deleted later if data issues resolved
```
Comment (Robinlovelace) [inline src/datasets.js:476]: :+1: OK commented out. data_zones is a function defined elsewhere then?

**Review by Robinlovelace — APPROVED** [review]
Looks fine. Say we want to add a different popup, would the code go in the same place? Grateful if you can point me to the code the creates the graphs in the popups.

File: src/datasets.js (hunk 1 above)
Comment (mvl22) [reply src/datasets.js:476]: Yes, to create auto-pops containing charts, just define a data structure as per the one I've just commented out, with the key as the layer name. Have added a quick comment in ab2e1f741d117baef8faf2ff4b4da8950f32cd9a.

The schools one is still present:
https://github.com/nptscot/nptscot.github.io/blob/dev/src/datasets.js#L584

**Comment by mvl22** [comment]
> Great to see, will look at code and merge. I imagine this involves removing lots of code, as with backend code 😌

No, no actual code is removed. It's just a deleted definition, so the popup won't be created.

**Pull request merged**